dipy/workflows/tests/test_io.py

```diff
@@ -342,3 +349,73 @@ def test_pam_to_niftis_flow():
         assert_true(os.path.isfile(flow.last_generated_outputs["out_shm"]))
         assert_true(os.path.isfile(flow.last_generated_outputs["out_gfa"]))
         assert_true(os.path.isfile(flow.last_generated_outputs["out_sphere"]))
+
+
+def test_math():
+    with TemporaryDirectory() as out_dir:
+        data_path, _, _ = get_fnames(name="small_101D")
+        data_path_a = pjoin(out_dir, "data_a.nii.gz")
+        data_path_b = pjoin(out_dir, "data_b.nii.gz")
+        shutil.copy(data_path, data_path_a)
+        shutil.copy(data_path, data_path_b)
+
+        data, _ = load_nifti(data_path)
+        operations = ["vol1*3", "vol1+vol2+vol3", "5*vol1-vol2-vol3", "vol3*2 + vol2"]
```

Comment:
Will this elegantly handle extraneous white space like "vol1 + vol2 "?

Reply:
yes, it handles it without problem. I did not put space because 88 line limit 😅

Reply:
I just realized that the last one on the list has a space between operator